pkg/endpoint/policy.go

```diff
@@ -707,8 +708,12 @@ func (e *Endpoint) regeneratePolicy(owner Owner, opts models.ConfigurationMap) (
 		"policyRevision.next": e.nextPolicyRevision,
 	}).Debug("Done regenerating")
 
-	// Return true if need to regenerate BPF
-	return optsChanged || policyChanged || e.nextPolicyRevision > e.policyRevision, consumersAdd, consumersRm, nil
+	// Need to regenerate BPF for this endpoint if any options for it have
+	// changed, // if the policy itself has changed, or the revision number for
```

Comment:
extra `//`, and also doesn't this comment just describe exactly what the line below says in code?

I like the separate `needToRegenerateBPF` variable though, I think that helps readability.

Reply:
You're right, I'll remove this comment. It was leftover from when I was going through and annotating the code to get understanding of it :)